projects/1.16/assets/apotheosis/apotheosis/lang/zh_cn.json

```
@@ -1,497 +1,581 @@
+    "jei.spw.title": "刷怪笼修改",
+    "jei.spw.invert": "反转调节器的行为",
+    "jei.spw.invert2": "必须用一只手拿着",
+    "jei.spw.editmindelay": "减小最小刷怪延迟",
+    "jei.spw.editmaxdelay": "减小最大刷怪延迟",
+    "jei.spw.editspawncount": "增加刷怪数量",
+    "jei.spw.editnearby": "增加附近最大实体数",
+    "jei.spw.editplayer": "增加玩家激活刷怪笼的范围",
+    "jei.spw.editspawn": "增加刷怪范围",
+    "jei.spw.ignoreconditions": "忽略刷怪条件",
+    "jei.spw.ignoreplayers": "忽略附近玩家",
+    "jei.spw.changeentity": "更改生成的实体",
+    "jei.spw.ignorecap": "忽略区域刷怪限制",
+    "jei.spw.redstone": "启用红石控制",
+    "jei.spw.instructions": "手持各种调节器右击刷怪笼可以调节刷怪笼的数据。\\n某个物品可以反转调节器的行为，导致反方向调节刷怪笼的数据。\\n使用精准采集挖掘可以获得刷怪笼。",
+    "jei.spw.capturing": "捕捉附魔可以使生物有概率在死亡时掉落它们的刷怪蛋。\\n每个附魔等级增加 0.4% 的几率。",
+	"jei.spw.minmax": "最小值：%s    最大值：%s",
+    "comment_id": "/附魔模块",
+    "block.apotheosis.hellshelf": "地狱书架",
+	"block.apotheosis.blazing_hellshelf": "炽热的地狱书架",
+	"block.apotheosis.glowing_hellshelf": "发光的地狱书架",
+	"block.apotheosis.seashelf": "海洋书架",
+	"block.apotheosis.crystal_seashelf": "澄澈的海洋书架",
+	"block.apotheosis.heart_seashelf": "锻心的海洋书架",
+	"block.apotheosis.endshelf": "末地书架",
+	"block.apotheosis.pearl_endshelf": "璀璨的末地书架",
+	"block.apotheosis.draconic_endshelf": "龙裔的末地书架",
+	"block.apotheosis.beeshelf": "蜜制书架",
+	"block.apotheosis.melonshelf": "西瓜书架",
+    "info.apotheosis.ench_power": "为附魔台提供 %s 的附魔能力。",
+    "info.apotheosis.cobweb": "可以用来去除所有附魔属性。",
+    "info.apotheosis.masterwork": "极品",
+    "info.apotheosis.twisted": "扭曲",
+    "info.apotheosis.corrupted": "腐化",
+    "info.apotheosis.enchanting": "附魔台现在可以超过 30 级。  一些附魔只会出现在 30 级以上的附魔台。",
+    "info.apotheosis.altar": "海之祭坛专注于转换附魔物品。\\n\\n首先，先在每根柱子上放置一个附魔物品，一旦提供了足够的物品，祭坛将吸取附近玩家的经验。\\n如果经验不足，将会产生紫色颗粒。\\n\\n一旦完成，海之祭坛上的物品会被消耗，并生成一本附魔书，有可能获得宝藏级附魔。",
+	"gui.apotheosis.enchant.eterna": "位阶",
+	"gui.apotheosis.enchant.quanta": "量子化",
+	"gui.apotheosis.enchant.arcana": "阿卡那",
+	"info.apotheosis.ench_stats": "附魔属性：",
+	"info.apotheosis.eterna": " - 位阶：%s",
+	"info.apotheosis.quanta": " - 量子化：%s%%",
+	"info.apotheosis.arcana": " - 阿卡那：%s%%",
+	"info.apotheosis.eterna.p": " - 位阶：+%s (最高 %s)",
+	"info.apotheosis.quanta.p": " - 量子化：+%s%%",
+	"info.apotheosis.arcana.p": " - 阿卡那：+%s%%",
+	"gui.apotheosis.enchant.eterna.desc": " 是强度的衡量标准。",
+	"gui.apotheosis.enchant.eterna.desc2": "它控制着附魔台的等级。",
+	"gui.apotheosis.enchant.eterna.desc3": "当前值：%s/%s",
+	"gui.apotheosis.enchant.quanta.desc": " 是强度波动的衡量标准。",
+	"gui.apotheosis.enchant.quanta.desc2": "它控制着附魔的变化幅度。",
+	"gui.apotheosis.enchant.quanta.desc3": "你最终获得的附魔等级将会因为此数值而产生变化。",
+	"gui.apotheosis.enchant.quanta.desc4": "当前值：%s%%",
+	"gui.apotheosis.enchant.arcana.desc": " 是质量的衡量标准。",
+	"gui.apotheosis.enchant.arcana.desc2": "它可以使非常罕见的附魔变得很常见，",
+	"gui.apotheosis.enchant.arcana.desc3": "而且它还能增加附魔的数量。",
+	"gui.apotheosis.enchant.arcana.desc4": "当前值：%s%%",
+	"ingredient.apotheosis.ench": "至少需要 %s",
+    "enchantment.apotheosis.hell_infusion": "炼狱灌注",
+    "enchantment.apotheosis.hell_infusion.desc": "在地狱里可以造成额外的魔法伤害。",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
    "enchantment.apotheosis.hell_infusion.desc": "在下界里可以造成额外的魔法伤害。",
```